fileutil/fileutil.go

```diff
@@ -89,3 +90,16 @@ func (fileManager) ensureSavePath(savePath string) error {
 func (f fileManager) WriteBytes(path string, value []byte) error {
 	return os.WriteFile(path, value, 0600)
 }
+
+// FileSizeInBytes checks if the provided path exists and return with the file size (bytes) using os.Lstat.
+func (fileManager) FileSizeInBytes(pth string) (float64, error) {
+	if pth == "" {
+		return 0, errors.New("No path provided")
+	}
+	fileInf, err := os.Lstat(pth)
+	if err != nil {
+		return 0, err
+	}
+
+	return float64(fileInf.Size()), nil
```

Comment:
FileInfo.Size is an int64, do we really need it as a float?

Reply:
Updated.